src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -325,13 +325,15 @@ protected void SetIntellisenseBuildResultAndNotifyWorkspaceHosts(bool succeeded)
                 {
                     m[ProjectGuidPropertyName] = Guid;
                     m[nameof(LastDesignTimeBuildSucceeded)] = LastDesignTimeBuildSucceeded;
-                    m[nameof(PushingChangesToWorkspaceHosts)] = PushingChangesToWorkspaceHosts;
+
+                    // Use old name for consistency
+                    m["PushingChangesToWorkspaceHosts"] = PushingChangesToWorkspace;
```

Comment:
I *just* added this too!

Reply:
Hence my annoyance when you merged this last week. :smile: